dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -164,10 +166,20 @@ public boolean isVerified() {
     }
 
     @ApiModelProperty(value = "Verified source for the version", position = 9)
+    @Deprecated
     public String getVerifiedSource() {
         return this.getVersionMetadata().verifiedSource;
     }
 
+    @ApiModelProperty(value = "Verified source for the version", position = 10)
```

Comment:
There is a position 10 on line 319 as well.

Reply:
:cry: